### PR TITLE
Add clojure extras plugin to docs

### PR DESCRIPTION
**Pull request by brcosta**
Please answer the following questions and leave the below in as part of your PR.

- [x] I have read the [developer documentation](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md).

- [ ] This PR correponds to an [issue with a clear problem statement](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [ ] This PR contains a [test](https://github.com/clj-kondo/clj-kondo/blob/master/doc/dev.md#tests) to prevent against future regressions

- [ ] I have updated the [CHANGELOG.md](https://github.com/clj-kondo/clj-kondo/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

File: doc/editor-integration.md [hunk 1]
```diff
@@ -359,3 +365,4 @@ and error counts on the message line:
 ```kak
 define-command -hidden -override lint-show-counters %{}
 ```
+::q!::q
```
Comment (borkdude) [inline doc/editor-integration.md:368]: @brcosta What is this? Have you been editing in vim? ;) Can you undo the changes that are not about clojure extras?

Comment (brcosta) [reply doc/editor-integration.md:368]: haha sorry man, will do
